server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaBuilder.java

```diff
@@ -55,9 +57,15 @@
     static final int DEFAULT_NUM_MIRRORING_THREADS = 16;
     static final int DEFAULT_MAX_NUM_FILES_PER_MIRROR = 8192;
     static final long DEFAULT_MAX_NUM_BYTES_PER_MIRROR = 32 * 1048576; // 32 MiB
-    static final String DEFAULT_CACHE_SPEC = "maximumWeight=134217728," + // Cache up to apx. 128-megachars
-                                             "expireAfterAccess=5m";      // Expire on 5 minutes of inactivity
+
     static final long DEFAULT_WEB_APP_SESSION_TIMEOUT_MILLIS = 604800000;   // 7 days
+    static final String DEFAULT_REPOSITORY_CACHE_SPEC =
+            "maximumWeight=134217728," + // Cache up to apx. 128-megachars.
+            "expireAfterAccess=5m";      // Expire on 5 minutes of inactivity.
+
+    static final String DEFAULT_SESSION_CACHE_SPEC =
+            // Expire after the duration of session timeout.
+            "maximumSize=4096,expireAfterWrite=" + (DEFAULT_WEB_APP_SESSION_TIMEOUT_MILLIS / 1000) + "s";
```

Comment:
Probably OK to use a larger value? Even if we assume a token takes 1 KiB, it will only be 4 MiB. How about 8192, which should cover almost all employees in most engineering organizations?